cdap-docs/build.sh

```diff
@@ -154,41 +150,43 @@ function copy_docs_lower_level() {
   echo "========================================================"
   echo ""
 
-  copy_html admin-manual
-  copy_html developers-manual
-  copy_html integrations
-  copy_html introduction
-  copy_html reference-manual
-  copy_html examples-manual
+  for i in ${MANUALS}; do
+    copy_html ${i}
+  done
 
   local project_dir
   # Rewrite 404 file, using branch if not a release
-  if [ "x$GIT_BRANCH_TYPE" == "xfeature" ]; then
-    project_dir=$PROJECT_VERSION-$GIT_BRANCH
+  if [ "x${GIT_BRANCH_TYPE}" == "xfeature" ]; then
+    project_dir=${PROJECT_VERSION}-${GIT_BRANCH}
   else
-    project_dir=$PROJECT_VERSION
+    project_dir=${PROJECT_VERSION}
   fi
-  rewrite $BUILD/$HTML/404.html "src=\"_static"  "src=\"/cdap/$project_dir/en/_static"
-  rewrite $BUILD/$HTML/404.html "src=\"_images"  "src=\"/cdap/$project_dir/en/_images"
-  rewrite $BUILD/$HTML/404.html "/href=\"http/!s|href=\"|href=\"/cdap/$project_dir/en/|g"
-  rewrite $BUILD/$HTML/404.html "action=\"search.html"  "action=\"/cdap/$project_dir/en/search.html"
+  rewrite ${BUILD}/${HTML}/404.html "src=\"_static"  "src=\"/cdap/${project_dir}/en/_static"
+  rewrite ${BUILD}/${HTML}/404.html "src=\"_images"  "src=\"/cdap/${project_dir}/en/_images"
+  rewrite ${BUILD}/${HTML}/404.html "/href=\"http/!s|href=\"|href=\"/cdap/${project_dir}/en/|g"
+  rewrite ${BUILD}/${HTML}/404.html "action=\"search.html"  "action=\"/cdap/${project_dir}/en/search.html"
 }
 
 ################################################## current
 
 function build_all() {
+  echo "========================================================================="
   echo "Building GitHub Docs."
-  ./build.sh docs-github $ARG_2 $ARG_3
+  echo "========================================================================="
+  run_command docs-github-part $ARG_2 $ARG_3
   echo "Stashing GitHub Docs."
   cd $SCRIPT_PATH
   mkdir -p $SCRIPT_PATH/$BUILD_TEMP
   mv $SCRIPT_PATH/$BUILD/*.zip $SCRIPT_PATH/$BUILD_TEMP
+  echo "========================================================================="
   echo "Building Web Docs."
-  ./build.sh docs-web $ARG_2 $ARG_3
+  echo "========================================================================="
+  run_command docs-web-part $ARG_2 $ARG_3
   echo "Replacing GitHub Docs."
   mv $SCRIPT_PATH/$BUILD_TEMP/*.zip $SCRIPT_PATH/$BUILD
   rm -rf $SCRIPT_PATH/$BUILD_TEMP
   bell
```

Comment:
You might want to make this an option. I have had complaints about this from..someone. Other people find it annoying. But it's very handy. So, put it in a function, and use an environment variable?